test/lib/autofix/snapshots/autofix.fixtures.ts.md

```diff
@@ -2310,3 +2241,1844 @@ Generated by [AVA](https://avajs.dev).
+    	const bundle = ResourceBundle.create({␊
+    		url: "resources/i18n.properties"␊
+    	});␊
+    	// Respect the context␊
+    	sap.ui.require(["sap/base/i18n/ResourceBundle"], function (innerScopeResourceBundle) {␊
+    		const bundle = innerScopeResourceBundle.create({␊
+    			url: "resources/i18n.properties"␊
+    		});␊
+    	});␊
+    ␊
+    	// https://github.com/SAP/ui5-linter/issues/522␊
+    	const logObject = Log.getLogger();␊
+    	const myLogger = Log.getLogger();␊
+    	const myLog = Log.getLogEntries();␊
+    	const myLog2 = Log.getLogEntries();␊
+    ␊
+    	const oLogListener = {};␊
+    	oLogListener.onLogEntry = function(oLog) {␊
+    		//␊
+    	};␊
+    	Log.addLogListener(oLogListener);␊
+    	Log.removeLogListener(oLogListener);␊
+    ␊
+    	const level = Log.getLevel();␊
+    	const logLevel = Log.Level;␊
+    	const isLoggable = Log.isLoggable(logLevel.DEBUG);␊
+    ␊
+    	Log.logSupportInfo(true);␊
+    	Log.debug("This is a debug log message");␊
+    	Log.error("This is a error log message");␊
+    	Log.fatal("This is a fatal log message");␊
+    	Log.info("This is a info log message");␊
+    	Log.trace("This is a trace log message");␊
+    	Log.warning("This is a warning log message");␊
+    ␊
+    	// https://github.com/SAP/ui5-linter/issues/524␊
+    	const myCSS = encodeCSS("+");␊
+    	const myJS = encodeJS("\\"");␊
+    	const myURL = encodeURL("a/b?c=d&e");␊
+    	const myURLParameters = encodeURLParameters({ a: true, b: "d e" });␊
+    	const myHTML = encodeXML("<p>My Text</p>");␊
+    	const myXML = encodeXML("<Text text=\\"MyText\\" />");␊
+    	␊
+    	// https://github.com/SAP/ui5-linter/issues/525␊
+    	URLListValidator.add("https", "example.com", 1337, "path");␊
+    	URLListValidator.clear();␊
+    	var aEntries = URLListValidator.entries();␊
+    	URLListValidator.validate("https://example.com");␊
+    	var jQuerySapRemoveUrlWhitelist = URLListValidator.entries();␊
+    jQuerySapRemoveUrlWhitelist.splice(0, 1);␊
+    URLListValidator.clear();␊
+    jQuerySapRemoveUrlWhitelist.forEach(({protocol, host, port, path}) => URLListValidator.add(protocol, host, port, path));␊
+    ␊
+    	// https://github.com/SAP/ui5-linter/issues/527␊
+    	var textCamelVase = camelize(" First Name Last ");␊
+    	var textUpperCase = capitalize("myValue");␊
+    	// capitalize does not have positional parameter, therefore we cannot migrate this case␊
+    	var textUpperCase1 = jQuery.sap.charToUpperCase("myValue", 5);␊
+    	var textUpperCase2 = capitalize("myValue");␊
+    	// Positions less than 0 or greater than the string length will capitalize the first letter␊
+    	var textUpperCase3 = capitalize("myValue");␊
+    	var textUpperCase4 = capitalize("myValue");␊
+    	const xy = "myValue";␊
+    	var textUpperCase2 = jQuery.sap.charToUpperCase(xy, 1);␊
+    	var textUpperCase2 = capitalize(xy); // Can't be migrated␊
+    	var textUpperCase2 = capitalize(someFunc()); // Can't be migrated␊
```

Comment:
I added these test cases. They shouldn't get migrated. It looks like in case of identifiers or expressions, the length of the first argument is incorrectly calculated on the length of the variable or function name.

Reply:
Outcome from team meeting: We don't want to migrate those at all as we don't know the type of the arguments